tests/investmentWithdrawal.test.ts

```
@@ -0,0 +1,207 @@
+        organizationId,
+        status: "requested",
+        amountAcbu,
+        availableAt: new Date(now.getTime() - 1000),
+      },
+    ]);
+
+    mockUpdate.mockResolvedValue({});
+
+    await processInvestmentWithdrawalAvailability();
+
+    expect(mockPublish).toHaveBeenCalledWith(null, amountAcbu.toNumber(), organizationId);
```

ESLint

[error] Replace `null,·amountAcbu.toNumber(),·organizationId` with `⏎······null,⏎······amountAcbu.toNumber(),⏎······organizationId,⏎····`

(prettier/prettier)